languageservice/src/context-providers/strategy.ts

```diff
@@ -1,49 +1,16 @@
 import {data, DescriptionDictionary} from "@actions/expressions";
-import {isMapping, isScalar, isString} from "@actions/workflow-parser";
-import {WorkflowContext} from "../context/workflow-context.js";
-import {scalarToData} from "../utils/scalar-to-data.js";
+import {getDescription} from "./descriptions.js";
 
-// Default strategy values when no strategy block is defined
-const DEFAULT_STRATEGY = {
-  "fail-fast": new data.BooleanData(true),
-  "job-index": new data.NumberData(0),
-  "job-total": new data.NumberData(1),
-  "max-parallel": new data.NumberData(1)
-};
-
-export function getStrategyContext(workflowContext: WorkflowContext): DescriptionDictionary {
```

Comment:
Deleted a bunch of logic. For autocomplete purposes all we need is a static config. The structure is always the same regardless whether it's a matrix job or not.